helix-front/client/app/shared/input-inline/input-inline.component.ts

```diff
@@ -88,7 +88,7 @@ export class InputInlineComponent implements ControlValueAccessor, OnInit {
     this.lastValue = value;
     this.editing = true;
     setTimeout((_) => {
-      this.inputControl.nativeElement.focus();
+      this.inputControl?.nativeElement.focus();
```

Comment:
I saw some errors in the console about this inputControl being ephemerally `undefined`, so I added this optional chaining operator `?.` to mitigate that situation.